manifests/g/GoLang/Go/1.22.11/GoLang.Go.installer.yaml

```
@@ -0,0 +1,53 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GoLang.Go
+PackageVersion: 1.22.11
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+InstallerSwitches:
+  InstallLocation: INSTALLDIR="<INSTALLPATH>"
+UpgradeBehavior: install
+Commands:
+- go
+- gofmt
+FileExtensions:
+- go
+ReleaseDate: 2025-01-16
+Installers:
+- Architecture: x86
+  InstallerUrl: https://go.dev/dl/go1.22.11.windows-386.msi
+  InstallerSha256: 1DCB84F20AFA980A0E8E5DAB6C2BCB0A6E5D21F22CB1AD965C960055906398E4
+  ProductCode: '{B98D8A82-C924-4AD2-89CA-6E9CDFF5F5C3}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Go Programming Language 386 go1.22.11
+    Publisher: https://go.dev
+    ProductCode: '{B98D8A82-C924-4AD2-89CA-6E9CDFF5F5C3}'
+    UpgradeCode: '{1C3114EA-08C3-11E1-9095-7FCA4824019B}'
+  InstallationMetadata:
+    DefaultInstallLocation: '%ProgramFiles(x86)%\Go'
+- Architecture: x64
+  InstallerUrl: https://go.dev/dl/go1.22.11.windows-amd64.msi
+  InstallerSha256: 8193C77C596453FAE1A67483923D8D3294DCC5E43DB2164F53C48E83ECA3F40F
+  ProductCode: '{65A21BAD-6460-4D21-A9AC-0180874BCC33}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Go Programming Language amd64 go1.22.11
+    Publisher: https://go.dev
+    ProductCode: '{65A21BAD-6460-4D21-A9AC-0180874BCC33}'
+    UpgradeCode: '{22EA7650-4AC6-4001-BF29-F4B8775DB1C0}'
+  InstallationMetadata:
+    DefaultInstallLocation: '%ProgramFiles%\Go'

```

```suggestion
    DefaultInstallLocation: '%ProgramFiles%\Go'
- Architecture: arm
  InstallerUrl: https://go.dev/dl/go1.22.11.windows-arm.msi
  InstallerSha256: DE65DCC4E44706ED4EAAAB248F71A28D7B634E8DE754313472554CD6CD49DCBA
  ProductCode: '{205E0F06-8455-4624-A66D-9488EB0F50C3}'
  AppsAndFeaturesEntries:
  - DisplayName: Go Programming Language arm go1.22.11
    Publisher: https://go.dev
    ProductCode: '{205E0F06-8455-4624-A66D-9488EB0F50C3}'
    UpgradeCode: '{22EA7650-4AC6-4001-BF29-F4B8775DB1C0}'
  InstallationMetadata:
    DefaultInstallLocation: '%ProgramFiles%\Go'
```

You should add the arm architecture installer to be consistent with the previous manifests.
(By komac analyse)